### ISSUE TITLE
[BUG] Snapshots do not match

### ISSUE THREAD

**Issue by ViceIce** [theramis/Snapper#50]
**Describe the bug**
On `CI=1` build the snapshot does not match. When update snapshot is set, snapshot does not change.

**To Reproduce**

Create `JObject` from Json:
```
{                                                                               
  "$id": "name=\"Excel\";itemId=\"2\";listId=\"Category\";type=\"Excel\";",     
  "Id": 2,                                                                      
  "CategoryName": "Condiments",                                                 
  "Description": "Sweet and savory sauces, relishes, spreads, and seasonings"   
}                                                                               
```

Add matcher
```cs
NUnit.Framework.Assert.That(obj, Snapper.Nunit.Is.EqualToChildSnapshot("test"));
```

Error:
```
  Error Message:
   "
Snapshots do not match
- Snapshot
+ Received


  "Description": "Sweet and savory sauces, relishes, spreads, and seasonings"
}
"
```

**Additional context**
Snapper Version: 2.2.1
Operation System: Windows Server 2019
.Net Version: net472, netcoreapp3.1
NUnit: 3.12.0

**Comment by theramis** [comment]
Thanks for the bug request. 
I think this was introduced as part of 2.2.1 sadly. 

Fixed in this PR https://github.com/theramis/Snapper/pull/51 . Will release as part of 2.2.2 soon 😄

**Comment by theramis** [comment]
@all-contributors please add @ViceIce  for bugs

**Comment by theramis** [comment]
Released as part of version 2.2.2!
